poll/check.go

```diff
@@ -0,0 +1,38 @@
+package poll
+
+import (
+	"net"
+	"os"
+)
+
+// Check is a function which will be used as check for the WaitOn method.
+type Check func(t LogT) Result
+
+// FileCheck looks on filesystem and check that the path exists.
+func FileCheck(path string) Check {
+	return func(t LogT) Result {
+		_, err := os.Stat(path)
+		if err != nil && os.IsNotExist(err) {
+			t.Logf("waiting on file %s to be available", path)
+			return Continue("file %s not available", path)
```

Comment:
To be consistent with terminology I think "available" should be change to "exists" as well.

"file %s to exist", "file %s does not exist"